mythril/ethereum/util.py

```diff
@@ -138,7 +139,20 @@ def solc_exists(version):
     elif sys.version_info[1] >= 6:
         # we are using solc-x for the the 0.5 and higher
         try:
-            return solcx.install.get_executable(version)
+            if platform.system() == "Darwin":
```

Comment:
This part can be made a function which can be reused in the mythril_disassembler.py right?

Reply:
Yeah I wasn't quite sure if that was used anywhere else, but yes effectively it's the same code twice. The code in mythril_disassembler.py throws exception if no version is found either on the system or for download